packages/pages/src/common/src/module/types.ts

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+/**
+ * The type definition for the module's default function.
+ *
+ * @public
+ */
+export type Module = () => React.JSX.Element;
+
+/**
+ * The exported `config` function's definition.
```

Comment:
Is the "function's" part a copy/paste from serverless functions? I would change this to be more meaningful for a user as this is an exposed type: "The configuration options for a Module."